invenio_records_rest/serializers/response.py

```diff
@@ -30,6 +30,7 @@ def view(pid, record, code=200, headers=None, links_factory=None):
         response.status_code = code
         response.set_etag(str(record.revision_id))
         response.last_modified = record.updated
+        response.vary = 'Content-Type, Accept'
```

Comment:
Not sure if anybody is relying on them, but I think it's better to use other `Accept-` headers too, especially `Accept-Language`

Reply:
I think since you provide `Accept` the `Content-Type` is redundant.
I would go with  `response.vary = 'Accept, Accept-Encoding, Accept-Language'`

Reply:
once this PR gets approved please use the headers like @topless is suggesting